src/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
You can remove the `eslint-disable`. I am okay with the line break here. Prettier does a good job of keeping things tidy.

Reply:
I didn't notice that :joy:  
I'll remove this line.